tests/unit/pynwb_tests/test_base.py

```diff
@@ -18,21 +18,21 @@ def test_init(self):
     def test_add_container(self):
         ts = TimeSeries('test_ts', [0, 1, 2, 3, 4, 5],
                         'grams', timestamps=[0.0, 0.1, 0.2, 0.3, 0.4, 0.5])
-        self.pm.add_container(ts)
+        self.pm.add_data_interface(ts)
```

Comment:
Should this test be renamed. The test is called ``test_add_container`` but it now tests ``add_data_interface`` instead. The same applies to the test_get_container case below.